pkg/products/fuse/reconciler.go

```diff
@@ -177,13 +177,21 @@ func (r *Reconciler) Reconcile(ctx context.Context, installation *integreatlyv1a
 		return phase, err
 	}
 
-	phase, err = r.reconcileTemplates(ctx, serverClient)
-	logrus.Infof("Phase: %s reconcileTemplates", phase)
+	// phase, err = r.reconcileTemplates(ctx, serverClient)
```

Comment:
Can we remove these commented lines? or are they kept intentionally?

Reply:
oops that should have been removed my mistake